### ISSUE TITLE
when I use the latest mmcls/models/losses/accuracy.py ,the topk is always 0.but the loss is decline

### ISSUE THREAD

**Issue by jfbiancheng** [open-mmlab/mmclassification#262]
![image](https://user-images.githubusercontent.com/45869870/118964352-fff03800-b999-11eb-8adc-ad8090dc3582.png)

**Comment by mzr1996** [comment]
Hello, can you provide the entire config file?
Generally, cross-entropy loss declines if the model output gets closer to the ground truth.
For example,
```python
>>> import torch
>>> import torch.nn.functional as F
>>> gt_label = torch.tensor([0])
>>> output1 = torch.tensor([[0.1, 0.1, 0.8]])
>>> print(F.cross_entropy(output1, gt_label))
tensor(1.3897)

>>> output2 = torch.tensor([[0.3, 0.3, 0.4]])
>>> print(F.cross_entropy(output2, gt_label))
tensor(1.1331)

```
Although all outputs pred is wrong, the cross-entropy still declines.

**Comment by mzr1996** [comment]
No update for a long time, closed.
If the problem is not solved, please open a new issue.
